modules/finance/src/test/java/com/opengamma/strata/finance/rate/bond/FixedCouponBondTradeTest.java

```diff
@@ -152,6 +163,7 @@ public void coverage() {
             .standardId(StandardId.of("Ticker", "GOV1-BND1"))
             .build()))
         .quantity(100L)
+        .payment(Payment.of(CurrencyAmount.of(EUR, NOTIONAL * 0.05), SETTLEMENT_DATE))
```

Comment:
Tests should have realistic values: payment amount = -NOTIONAL * QUANTITY * 0.99 or something similar.